core/include/drivers/ls_dspi.h

```diff
@@ -27,6 +27,13 @@
 #define DSPI_CTAR_A_DT(x)    (((x) & 0x0F) << 4)	/* DT [24-27] */
 #define DSPI_CTAR_BR(x)	     ((x) & 0x0F)	/* Baud Rate Scaler [28-31] */
 
+/* SPI mode flags */
```

Comment:
Why are these needed here?

Reply:
They are used in platform specific SPI configuration function.

Reply:
Not upstream, but not a big deal either. Please put this change in a separate commit explaining why it's needed.

Reply:
Done.